Extensions/xkit_patches.js

```diff
@@ -1,32 +1,25 @@
 //* TITLE XKit Patches **//
-//* VERSION 7.1.2 **//
+//* VERSION 7.1.3 **//
 //* DESCRIPTION Patches framework **//
 //* DEVELOPER new-xkit **//
 
 XKit.extensions.xkit_patches = new Object({
 
 	running: false,
 
-	preferences: {
-		debug_mode: {
-			text: "XKit Developer Mode",
-			default: true,
-			value: true
-		}
-	},
-
 	run: function() {
 		this.running = true;
 
 		var to_run = [];
-		for (var i in this.patches) {
-			to_run.unshift(i);
-			if (i === XKit.version) {
+		for (let i of this.run_order.reverse()) {
+			to_run.push(i);
```

Comment:
why push (and reverse later) instead of unshift?

Reply:
just made more sense in my head, but if thats just me being weird ill gladly put it back to unshift